Extensions/messaging_tweaks.js

```diff
@@ -322,6 +351,14 @@ XKit.extensions.messaging_tweaks = new Object({
 		}
 		$(".true-icon").removeClass("true-icon");
 		$(".tab.iconic.tab_messaging").show();
+		$(".xkit-others_messaging_message, .xkit-my_messaging_message").each(function() {
+			var msg_div = $(this).find(".message-bubble .message");
+			msg_div.find("img").each(function() {
+				var img_html = $('<div>').append($(this).clone()).html();
+				var img_alt = $(this).attr("alt");
+				msg_div.html(msg_div.html().replace(img_html, img_alt.replace(new RegExp("꞉", "gi"), ":")));
```

Comment:
`img_alt.replace(new RegExp("꞉", "gi"), ":"))` looks like it doesn't do anything, is there something it is meant to do?